appengine/flexible/django_cloudsql/mysite/settings.py

```diff
@@ -81,7 +81,7 @@
 # [START dbconfig]
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.mysql',
```

Comment:
I think it's probably useful to keep the mysql one around and add a comment that if you're using Cloud SQL for MySQL you should use that engine instead.